inmaputil/cmd.go

```diff
@@ -569,6 +575,33 @@ func init() {
 	Root.AddCommand(srCmd)
 	srCmd.AddCommand(srPredictCmd)
 	Root.AddCommand(workerCmd)
+
+	// For some reasons, HTTPPort flag never seem to be passed down?
```

Comment:
This is because the flags are only evaluated when the cobra.command runs, not during global initialization. 

I think starting the GUI server during global initialization is causing the tests to fail. (You can see this in the test log output accompanying the pull request [the red stuff at the bottom of the conversation page) and also if you run `go test` in the inmaputil directory.

It also forces the user to use the GUI, they can no longer run InMAP from the command line. I think it would be better to start the server in `Root.Run()`. That way, the server only starts if the root command runs, which only happens if the user doesn't specify any subcommands at the command line. If the user specifies subcommands on the command line, then it would just run on the command line.

It would be a little tricky to do this, but I think it would work if you do it like this: https://play.golang.org/p/EEg5GAvQTv

Reply:
I actually initially started with gobra being inside Root.Run, but I ran into issues about circularity. I will think about this.